package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "phonegap-plugin-barcodescanner",
-  "version": "7.0.0",
+  "name": "@fttx/phonegap-plugin-barcodescanner",
+  "version": "7.0.1",
```

Comment:
Revert this change. The version number of the plugin will be updated when it is published to npm.